cli/src/commands/account.rs

```diff
@@ -389,11 +414,75 @@ mod tests {
         );
 
         let vanity = None;
-        let account = Account::New { network: 0, seed, vanity, discreet: false };
+        let account = Account::New { network: 0, seed, vanity, discreet: false, save_to_file: None };
         let actual = account.parse().unwrap();
         assert_eq!(expected, actual);
     }
 
+    #[test]
+    fn test_new_save_to_file() {
+        let dir = TempDir::new().expect("Failed to create temp folder");
+        let file = dir.path().join("my-private-key-file").to_string_lossy().to_string();
+
+        let seed = Some(1231275789u64.to_string());
```

Comment:
```suggestion
        let seed = Some("1231275789u64");
```
nit: most of these test strings don't need to be heap-allocated `String` objects, they could all just be literals

Reply:
The parameter needs to be `Some(String)` this suggestion doesn't work in this exact example. However, great find and I will remember it in future changes.